util/src/main/java/com/google/cloud/hadoop/util/BaseAbstractGoogleAsyncWriteChannel.java

```diff
@@ -242,7 +242,7 @@ private InputStream initializeUploadPipe() throws IOException {
    *
    * @throws IOException on IO error
    */
-  private T waitForCompletionAndThrowIfUploadFailed() throws IOException {
+  protected T waitForCompletionAndThrowIfUploadFailed() throws IOException {
```

Comment:
Do we need this change ?

Reply:
yes. it is used in `GoogleCloudStorageGrpcWriteChannel.java`